core/domain/html_validation_service.py

```diff
@@ -807,8 +807,7 @@ def get_filename_with_dimensions(old_filename, exp_id):
     file_system_class = (
         fs_domain.ExplorationFileSystem if feconf.DEV_MODE
         else fs_domain.GcsFileSystem)
-    fs = fs_domain.AbstractFileSystem(file_system_class(
-        'exploration/%s' % exp_id))
+    fs = fs_domain.AbstractFileSystem(file_system_class(exp_id))
```

Comment:
Just curious -- Why this change? (I don't have much knowledge here)

Reply:
It was this only when we ran the Job on testserver, then there was some other error that was supposed to be corrected. That was to be handled separately, but i mistakenly changed this.

Reply:
Okay gotcha, thanks!